Rakefile

```diff
@@ -1,7 +1,7 @@
 require "logstash/devutils/rake"
 
 desc "Get jars"
-task :get_jars do
+task :vendor do
```

Comment:
I'm OK with this change as-is, but in the future, we should refactor this to use the devutils `rake vendor` (it supports file unpacking and stuff, iirc)